guide/Text/Embed/example.md

```diff
@@ -45,47 +48,69 @@ $selectMenu[id;placeholder;min value(optional);max value;(optional);label;desc;v
 {deletecommand}
 {timestamp}
 {pin}
+```
 
 ### Only for interactions
 
+```
 {ephemeral=yes/no}
+{interaction}
+{return_id=yes/no}
```

Comment:
what is return_id? it doesn't belong to curl message

Reply:
idk, it was there previously